k3s-io/k3s
# [Release-1.25] Enhance `check-config`

#### Proposed Changes ####
Backport:
- https://github.com/k3s-io/k3s/pull/6946
- https://github.com/k3s-io/k3s/pull/7091


#### Types of Changes ####



#### Verification ####
Run 

#### Verification ####
- Remove Iptables from a host
- Install K3s
- Run `k3s check-config`, validate that something like:
```
System:
- /var/lib/rancher/k3s/data/e41fb3fc61cae853bd497dcbe628059f2fd8f196745962a7bec3dec999149452/bin/aux iptables v1.8.8 (legacy): ok
```
Should shows up. 
- `CONFIG_CGROUP_PIDS` `NETFILTER_XT_MATCH_COMMENT` and `NETFILTER_XT_MATCH_MULTIPORT` are now under `Generally Necessary:`


#### Testing ####




#### Linked Issues ####
https://github.com/k3s-io/k3s/issues/7119
https://github.com/k3s-io/k3s/issues/7163


#### User-Facing Change ####

```release-note

```

#### Further Comments ####
